requirements.txt

```diff
@@ -1,2 +1,3 @@
 pyTelegramBotAPI~=4.4.0
 requests~=2.27.1
+Pillow>=7.1.0  # `PIL.ImageGrab` for Linux
```

Comment:
The comment is unclear: `Pillow` is for `PIL.ImageGrab`; `>=7.1.0` is for "for Linux"